docs/tutorial/02_deploying_magma_orchestrator.md

```diff
@@ -0,0 +1,255 @@
+    nginx-proxy: a7b65619cbf8443feb05910823b6c42b-748407990.us-east-2.elb.amazonaws.com
+    orc8r-bootstrap-nginx: ab4feaf807c9440b8b021395f0422b26-1385734990.us-east-2.elb.amazonaws.com
+    orc8r-clientcert-nginx: ac63855b699064f5b825e707cfe290f0-1144570992.us-east-2.elb.amazonaws.com
+    orc8r-nginx-proxy: a6ae77105db624f06bb37f18843db925-911888461.us-east-2.elb.amazonaws.com
+  status: completed
+  timing:
+    completed: 2023-03-08 19:26:46 +0000 UTC
+    enqueued: 2023-03-08 19:26:44 +0000 UTC
+    started: 2023-03-08 19:26:45 +0000 UTC
+```
+
+The hostnames associated to each service will differ from those shown here.
+
+Create a file named `dns.json` with the following content:
+
+```json title="dns.json" hl_lines="7 12 20 25 33 38 46 51"
+{
+  "Comment": "CREATE CNAME records",
+  "Changes": [
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "bootstrapper-controller.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "ab4feaf807c9440b8b021395f0422b26-1385734990.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "api.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "a6ae77105db624f06bb37f18843db925-911888461.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "controller.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "ac63855b699064f5b825e707cfe290f0-1144570992.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "*.nms.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "a7b65619cbf8443feb05910823b6c42b-748407990.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+Each <mark>highlighted line</mark> line needs to be modified.
+
+!!! note
+    Replace each resource record value with the ones received from the previous step using the following Service/Hostname scheme:
+
+    | Service                         | Hostname                                     | 
+    |---------------------------------|----------------------------------------------|
+    | `<orc8r-bootstrap-nginx FQDN>`  | `bootstrapper-controller.<your domain name>` | 
+    | `<orc8r-nginx-proxy FQDN>`      | `api.<your domain name>`                     | 
+    | `<orc8r-clientcert-nginx FQDN>` | `controller.<your domain name>`              | 
+    | `<nginx-proxy FQDN>`            | `*.nms.<your domain name>`                   | 
```

Comment:
How about giving this note before the json example? 
Reading it for the first time, when I saw the json example, my first thought was `Maybe we should replace the FQDNs with references to the output of the get-load-balancer-addresses command?`. Then I scrolled a little bit and here it is. If this note was placed before the example, then the user won't have to figure out which value in the json corresponds to which value from the command output?

Reply:
Fixed, let me know if this is better

Reply:
Yeah, I think it's cool.